src/runtime/pkg/device/config/config.go

```diff
@@ -320,6 +331,9 @@ type VhostUserDeviceAttrs struct {
 	CacheSize uint32
 
 	QueueSize uint32
+
+	// Reconnect timeout for socket of vhost user block device
+	ReconnectTime uint32
```

Comment:
Nit: change to `ReconnectTimeout`.

And suggest adding a `timeout` suffix to other var names to let readers know that it is a timeout but not a reconnect count.

Reply:
ok